src/components/common/ListItem.js

```diff
@@ -59,15 +59,30 @@ type Props<IDType> = {
    * @desc Icon base64 image to be displayed on the left
    */
   iconSource?: Image.propTypes.source,
-}
+  /**
```

Comment:
You don't need to add this, since there is already a way to render something on the right of ListItem. You can use AdditionalView prop for that.

Reply:
Changed.